utils_test.go

```diff
@@ -145,4 +146,38 @@ func testUtils(t *testing.T, when spec.G, it spec.S) {
 			}
 		})
 	})
+
+	when(".ResolveRunImage", func() {
```

Comment:
Please see https://github.com/buildpacks/lifecycle/pull/617/files#r645779179.

Reply:
If we deleted this `.` here, I think it's worth removing the `.` in the other `when` in this file.
* ReadOrder
* ReadGroup
* WriteTOML
* TruncateSha

Reply:
Do we want to remove `.`? I did it because it was requested, but I also am fine with leaving them and matching the style of the test that already exist. That is why I used the `.` in the first place...

Reply:
I'm not sure. I think this is the only file that is using `.` and I'm not sure why.
Is there a reason for it?

Reply:
🤷  - I've seen this pattern before for denoting public method tests. `#Method` or `.Method`

I am happy to remove them all or match the style of the file

Reply:
But we're usually testing public methods and I have never seen this style anywhere else, but maybe I'm wrong and I just missed it.
Anyway, my first thought was about removing it but I don't have a strong opinion about it.

Reply:
Just curious, when do we use `.` vs `#`? (I've seen "object method" vs "static method" in other languages but I'm not sure what is our convention.) That said I don't have an opinion here.